pages/docs/installation/installation-packages.md

```diff
@@ -42,7 +42,21 @@ These packages are maintained by the preCICE community and may be occasionally o
 However, we appreciate the effort and you may be able to contribute to them.
 
 - [MSYS2](https://packages.msys2.org/base/mingw-w64-precice) (for Windows, built with MinGW), [thread on our forum](https://precice.discourse.group/t/precice-and-mingw-packages/382)
-- [Nix / NixOS](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&query=precice)
 - [EasyBuild](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/p/preCICE)
 - [Conda](https://anaconda.org/conda-forge/precice) (see also packages [pyprecice](https://anaconda.org/conda-forge/pyprecice) and [fenicsprecice](https://anaconda.org/conda-forge/fenicsprecice))
 - [FreeBSD](https://www.freshports.org/science/precice)
+
+### Nix / NixOS
```

Comment:
We've had a discussion about this. The sections above are all officially maintained, while this project is more a community effort. I can, however, also place it up there since it is a bit more text than the other community efforts.

Reply:
Since these packages are now in the preCICE organization, I think we will keep maintaining them in some way. Community efforts are completely out of our hands.